tap_stripe/__init__.py

```diff
@@ -107,7 +107,7 @@
 
 # NB: These streams will only sync through once for creates, never updates.
 IMMUTABLE_STREAMS = {'balance_transactions', 'events'}
-IMMUTABLE_STREAM_LOOKBACK = 300 # 5 min in epoch time, Stripe accuracy is to the second
+IMMUTABLE_STREAM_LOOKBACK = 600 # 10 min in epoch time, Stripe accuracy is to the second
```

Comment:
Add Jira ticket in the comment for this change.

Reply:
Added the jira ticket in the comment